Command/GifOptimizerCommand.php

```diff
@@ -26,7 +26,7 @@ class GifOptimizerCommand extends Command
     /**
      * @var string
      */
-    const DEFAULT_OPTIMIZATION_LEVEL = '-O3';
+    const DEFAULT_OPTIMIZATION_LEVEL = '-O1';
```

Comment:
This hasn't looked to have made much difference. Don't have time now but may have a look at tweaking some more settings. It could also just be that the video that which the gif came from was shot in low light using a basic camera and thus optimization of it is tricky.